content/rc/security/single-sign-on/saml-sso/saml-integration-okta-generic.md

```diff
@@ -0,0 +1,291 @@
+
+### Activate SAML in Access Management
+
+To activate SAML, you must have a local user (or social sign-on user) with the **owner** role. If you have the correct permissions, you will see the **Single Sign-On** tab.
+
+1. Fill in the information you saved in [step 6 previously](#step6) in the **setup** form. This includes :
+
+    * **Identity Provider Single Sign-On URL:**
+    * **Identity Provider Issuer**
+    * **X.509 Certificate**
+
+    {{<image filename="images/rc/saml/sm_saml_1.png" alt="" >}}{{</image>}}
+
+    {{<image filename="images/rc/saml/sm_saml_2.png" alt="" >}}{{</image>}}
+
+2. Once you click the **enable** button, wait a few seconds for the status to change. You will then be able to **download** the service provider (SP) metadata. Save the file to your local hard disk.
+
+    {{<image filename="images/rc/saml/sm_saml_3.png" alt="" >}}{{</image>}}
+
+3. Open the file in any text editor, and there are 2 pieces of information that you need to copy and mark down:
+
+    * **EntityID** : The unique name of the service provider (SP)
+
+    {{<image filename="images/rc/saml/sm_sam1_4.png" alt="" >}}{{</image>}}
+
+    * **Location** : The location of the assertion consumer service
+
+    {{<image filename="images/rc/saml/sm_sam1_5.png" alt="" >}}{{</image>}}
+
+4. Return to you OKTA application, **click on Applications -> Redis Cloud -> General** and click the **Edit** link
+
+    {{<image filename="images/rc/saml/okta_saml_app_int_12.png" alt="" >}}{{</image>}}
+
+5. Then, navigate to the **Configure SAML** (step 2) and update the following information :
+
+    * SAML Settings General
+
+        * **Single sign on URL** : Use the information that you copied for **Location**
+        * **Audience URI (SP Entity ID)** : Use the information that you copied for **EntityID**
+        * **Default RelayState** : **Only needed if you want to have an IdP initiated flow**. Take the ID from the location URL and append to the url (ex: https://app.redislabs.com/#/login/?idpId=YOUR_LOCATION_ID)
+
+    {{<image filename="images/rc/saml/okta_saml_app_int_13.png" alt="" >}}{{</image>}}
+
+    Once you have entered the information, click on the **Next** button, and finally click on the **Finish** button in feedback step.
+
+### Return to Redis Cloud SM
+
+1. Return to Redis Cloud SM, and now click on the **Activate** button
+
+    {{<image filename="images/rc/saml/sm_saml_8.png" alt="" >}}{{</image>}}
+
+2. A popup will appear, explaining that in order to test the SAML connection, that we need to login with OKTA credentials of a user defined in the Redis Cloud group. This is a user who is part of the group that we assigned the Redis Cloud application to.
+
+    {{<image filename="images/rc/saml/sm_saml_9.png" alt="" >}}{{</image>}}
+
+3. The OKTA login screen will appear, enter the credentials and click **Sign In**
+
+    {{<image filename="images/rc/saml/sm_saml_14.png" alt="" >}}{{</image>}}
```

Comment:
@ggeorges2015 It looks like this image is still broken. I don't see this image's filename in the list of added files, so maybe it was never checked in.